docs/topics/proc_creating-an-application.adoc

```diff
@@ -43,9 +43,120 @@ include::resources/vert-x/vertx-starter-example-pom.xml[]
 ----
 endif::[]
 ifdef::built-for-spring-boot[]
+ifndef::built-for-spring-boot-1-5[]
+include::note_spring-boot-2-supported-builder-images.adoc[]
++
+endif::built-for-spring-boot-1-5[]
 [source,xml,options="nowrap",subs="attributes+"]
 ----
-include::resources/spring-boot/spring-boot-starter-example-pom.xml[]
+\include::resources/spring-boot/spring-boot-starter-example-pom.xml[]
```

Comment:
The backwards slash doesn't look right.

Reply:
The backward slash disables the include directive.
This is done so that selected sections in the `pom.xml` example can be conditionalized. conditionals do not get processed if they are in a file that itself is included. this was meant as a workaround to avoid creating 2 example pom files.

Reply:
OK then, thanks for the info.